data/ecosystems/c/celo.toml

```diff
@@ -893,7 +995,7 @@ url = "https://github.com/celo-org/build-with-celo-hackathon"
 
 [[repo]]
 url = "https://github.com/celo-org/celo-blockchain"
-tags = ["Protocol"]
+tags = [ "Protocol" ]
```

Comment:
🔥

Reply:
Used a new tool to convert from TOML to JSON and back.